tests/MSBuildDeviceIntegration/Resources/LinkDescTest/HttpClientTest.cs

```diff
@@ -1,19 +1,24 @@
 using System;
+using System.Text;
 using System.Net.Http;
 
 public class HttpClientTest
 {
 	// [Test]
-	public static string NewHttpClient()
+	public static string Post()
```

Comment:
If the line is going to be changed, please add a ` ` (space) between the method name and the `()`.